Release Godot 3.5.3 and publish release notes (#720)
godotengine/godot-website: collections/_article/maintenance-release-godot-3-5-3.md

---
title: "Maintenance release: Godot 3.5.3"
excerpt: "It's been way too long since our previous 3.5 maintenance release! This new Godot 3.5.3 fixes a number of important issues and adjusts to changing platform requirements."
categories: ["release"]
author: Rémi Verschelde
image: /storage/blog/covers/maintenance-release-godot-3-5-3.webp
image_caption_title: Hauma
image_caption_description: A visual novel by SenAm Games
date: 2023-09-25 14:00:00
---

Long time no see, Godot 3.5! While our development focus since [Godot 4.0](/article/godot-4-0-sets-sail/) in March has been to release new 4.x versions at a steady pace (with [Godot 4.1](/article/godot-4-1-is-here/) in July, and Godot 4.2 scheduled for November), we're not forgetting about users who are still using Godot 3.5.

> Everyone seemed pretty happy with the [3.5.2 release](/article/maintenance-release-godot-3-5-2/), and it's the one most 3.x games have been published with in the past few months. But there's still a number of issues that are worth addressing in that branch, including notably some improvements for platform support such as targeting Android 13 by default. Also, we're still working on the upcoming Godot 3.6 feature release, which should soon reach the release candidate stage.

Reviewer: ```suggestion
Everyone seemed pretty happy with the [3.5.2 release](/article/maintenance-release-godot-3-5-2/), and it's the one most 3.x games have been published within the past few months. But there are still a number of issues that are worth addressing in that branch, including notably some improvements for platform support such as targeting Android 13 by default. Also, we're still working on the upcoming Godot 3.6 feature release, which should soon reach the release candidate stage.
```

> Everyone seemed pretty happy with the [3.5.2 release](/article/maintenance-release-godot-3-5-2/), and it's the one most 3.x games have been published with in the past few months. But there's still a number of issues that are worth addressing in that branch, including notably some improvements for platform support such as targeting Android 13 by default. Also, we're still working on the upcoming Godot 3.6 feature release, which should soon reach the release candidate stage.

Reviewer 2: This sentence feels weird.
> and it's the one most 3.x games have been published with in the past few months

```suggestion
Everyone seemed pretty happy with the [3.5.2 release](/article/maintenance-release-godot-3-5-2/). For the matter, that release has seen most 3.x games being published with it in the past few months. But there's still a number of issues that are worth addressing in that branch, including notably some improvements for platform support such as targeting Android 13 by default. Also, we're still working on the upcoming Godot 3.6 feature release, which should soon reach the release candidate stage.
```

Reviewer 3: I don't find it weird. 🤔

Author: It's a bit convoluted I guess.
